src/core/Akka.Remote/EndpointManager.cs

```diff
@@ -643,7 +643,7 @@ protected override SupervisorStrategy SupervisorStrategy()
                     });
 
                 return directive;
-            });
+            }, false);
```

Comment:
I turned off the default logging on OneForOneStrategy - basically I followed the guide of this class. It's written there: you can turnoff the logging and do logging yourself on the Decider implementation